chapters/operatorsandexpressions.tex

```diff
@@ -393,29 +393,34 @@ \subsection{Built-in Variable time}\label{built-in-variable-time}\indexinline{ti
 \end{lstlisting}
 \end{example}
 
-\section{Built-in Intrinsic Operators with Function Syntax}\label{built-in-intrinsic-operators-with-function-syntax}
+\section{Built-in Operators and Functions}\label{built-in-intrinsic-operators-with-function-syntax}
 
-Certain built-in operators of Modelica have the same syntax as a function call.
+Certain built-in operators of Modelica are called using the same syntax as a function call.
 However, they do not behave as a mathematical function, because the result depends not only on the input arguments but also on the status of the simulation.
 
 There are also built-in functions that depend only on the input argument, but also may trigger events in addition to returning a value.
-Intrinsic means that they are defined at the Modelica language level, not in the Modelica library.
-The following built-in intrinsic operators/functions are available:
+The built-in functions may also be overloaded such that a single Modelica function cannot be compatible with all calls of the function.
+Here, \emph{built-in} means that they are defined at the Modelica language level, not through a Modelica \lstinline!function! definition.
+The following built-in operators/functions are available:
 \begin{itemize}
 \item
-  Mathematical functions and conversion functions, see \cref{numeric-functions-and-conversion-functions} below.
+  Mathematical functions and conversion operators, see \cref{numeric-functions-and-conversion-functions} below.
 \item
   Derivative and special purpose operators with function syntax, see \cref{derivative-and-special-purpose-operators-with-function-syntax} below.
 \item
-  Event-related operators with function syntax, see \cref{event-related-operators-with-function-syntax} below.
+  Event-related operators, see \cref{event-related-operators-with-function-syntax} below.
 \item
-  Array operators/functions, see \cref{array-dimension-lower-and-upper-index-bounds}.
+  Array functions, see \cref{built-in-array-functions}.
```

Comment:
Many of the built-ins for arrays are operators.  For instance, a function couldn't describe the result type of `promote`, and `ndims` and `size` don't follow the variability rules for functions.  If anything, I think that it is the title of the section that should be changed (and that some of the definitions in the chapter should use `functiondefinition` instead of `operatordefinition`).
```suggestion
  Array operators/functions, see \cref{built-in-array-functions}.
```